packages/libs/react-ui/src/components/Stack/Stack.tsx

```diff
@@ -35,6 +43,14 @@ export const Stack = ({
   alignItems = undefined,
   wrap = undefined,
   direction = undefined,
+  width = undefined,
+  padding = undefined,
+  paddingX = undefined,
+  paddingY = undefined,
+  paddingTop = undefined,
+  paddingBottom = undefined,
+  paddingLeft = undefined,
+  paddingRight = undefined,
```

Comment:
Is this necessary? I would expect margin to be sufficient

Reply:
I think it's good to allow `padding` as well, personally prefer padding instead of margin to avoid vertical margin collapsing issues.

Reply:
One of the implementations of the `<Stack>` component was applying their own styles to it. The other properties could be solved with the existing code but padding was left out